qbraid/runtime/origin/job.py

```
@@ -0,0 +1,138 @@
+        """Cancel the OriginQ job. Not supported by the QCloud SDK."""
+        raise OriginJobError("OriginQ does not support job cancellation.")
+
+    def result(self) -> Result:
+        """Return the result of the OriginQ job."""
+        self.wait_for_final_state()
+        backend_job = self._get_backend_job()
+
+        try:
+            result_obj = backend_job.result()
+        except Exception as exc:
+            raise OriginJobError(f"Failed to fetch results for OriginQ job {self.id}") from exc
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)